arm/Microsoft.RecoveryServices/vaults/deploy.bicep

```diff
@@ -126,43 +126,34 @@ resource rsv 'Microsoft.RecoveryServices/vaults@2021-08-01' = {
   properties: {}
 }
 
-module rsv_backupStorageConfig 'backupStorageConfig/deploy.bicep' = {
+module backupStorageConfiguration 'backupStorageConfig/deploy.bicep' = {
   name: '${uniqueString(deployment().name, location)}-BackupStorageConfig'
   params: {
     recoveryVaultName: rsv.name
-    vaultStorageType: backupStorageConfig.vaultStorageType
-    enableCRR: backupStorageConfig.enableCRR
+    storageModelType: backupStorageConfig.storageModelType
+    crossRegionRestoreFlag: backupStorageConfig.crossRegionRestoreFlag
   }
-  dependsOn: [
-    rsv
-  ]
 }
 
-module rsv_protectionContainers 'protectionContainers/deploy.bicep' = [for (protectionContainer, index) in protectionContainers: {
+module protectionContainer 'protectionContainers/deploy.bicep' = [for (protectionContainer, index) in protectionContainers: {
```

Comment:
Please change back to `rsv_protectionContainers`

Reply:
Done!